test/setup.ts

```diff
@@ -36,11 +37,20 @@ use(sinonChai);
 use(chaiSorted);
 use(deepEqualInAnyOrder);
 
+config();
 process.env.NODE_ENV = 'test';
-process.env.HTTP_PORT = '3001';
-process.env.TYPEORM_CONNECTION = 'sqlite';
-process.env.TYPEORM_DATABASE = ':memory:';
-process.env.TYPEORM_SYNCHRONIZE = 'true';
+if (!process.env.TYPEORM_CONNECTION) {
```

Comment:
This execution flow needs to be tweaked somehow. Because in a development setting, you probably have your own "production" database in a SQLite file while you still write your test cases. You want to run these test cases 90% of the time from an in-memory SQLite database, but only use an actual database file when you want to debug a test case. I think the database error when running a test locally is very confusing, because it is not entirely clear what database is being used.